account-kit/react/src/createConfig.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  createConfig as createCoreConfig,
+  type AlchemyAccountsConfig,
+  type CreateConfigProps,
+} from "@account-kit/core";
+import type { AlchemyAccountsUIConfig } from "./types";
+
+export type AlchemyAccountsConfigWithUI = AlchemyAccountsConfig & {
+  ui?: AlchemyAccountsUIConfig;
+};
+
+/**
+ * Wraps the `createConfig` that is exported from `@aa-sdk/core` to allow passing
+ * an additional argument, the configuration object for the Auth Components UI
+ * (the modal and AuthCard).
+ *
```

Comment:
nit: if you add an `@example` tag here the docs will also have a usage section added to it